routes/api/v1/meals.js

```diff
@@ -20,4 +21,27 @@ router.get("/", function(req, res, next) {
   });
 });
 
+/*DELETES a food from meal*/
+router.delete("/:id/foods/:food_id", function(req, res, next) {
+  return MealFood.findOne({
```

Comment:
Actually wouldn't we want to findAll just in case there were multiple? There shouldn't be because of that fancy double-index thing we did, but I think it'd be more thorough to do a findAll.

Reply:
Nevermind, works fine as is.